src/main/treeDataProviders/issuesTree/codeFileTree/codeIssueTreeNode.ts

```diff
@@ -24,4 +20,11 @@ export abstract class CodeIssueTreeNode extends IssueTreeNode {
     public get regionWithIssue(): vscode.Range {
         return this._regionWithIssue;
     }
+
+    // Analyzer locations minimum is 1, vscode positions starts at 0
```

Comment:
Please elaborate on the comment - what's the "Analyzer locations"? the location coming back from the SARIF?

and BTW if it's minimum is 1 - when would we get 0?

Reply:
There is a different mapping that vscode expects as positions at file. 
For vscode the minimum value (i.e first row/col) is 0 while the results mapped to it is 1 from the scanners.

I suppose that the new scanners (CA and secrets) can have values of 0 like we are getting back from a sample project:
```
"region": {
      "snippet": {
             "text": "*******"
      },
      "endColumn": 0,
      "endLine": 3,
      "startColumn": 0,
      "startLine": 3
},
```